eventing/default-channels.md

```diff
@@ -0,0 +1,129 @@
+# Default Channels
+
+The default channel configuration allows channels to be created without
+specifying a provisioner. This leaves the selection of channel provisioner and
+properties up to the operator. The operator controls the default settings via a
+ConfigMap.
+
+## Creating a default channel
+
+To create a default channel, use the same procedure as for creating a normal
+channel, but leave the `spec.provisioner` property blank. The `spec` property
+must be provided, but should be empty.
```

Comment:
Why does spec need to be provided, rather than left empty?

Reply:
I think this is required due to the way go's json patch works. If the spec is omitted, it's set to nil instead of an empty map, but the patch assumes the map already exists. @Harwayne is this accurate?